README.md

```diff
@@ -1,70 +1,22 @@
-# Getting Started with Create React App
+# Work with API
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Deployed Link
+[Click Here](https://elaborate-taffy-18d03c.netlify.app/)
 
-## Available Scripts
 
-In the project directory, you can run:
+## Technologies
+Project is created with:
+* React : version 18.2.0
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Setup
+To run this project, install it locally using npm:
 
-### `npm test`
+```
+$ cd ../api
```

Comment:
This readme is misleading, why do you cd into api folder?

Reply:
if you're not currently in the project folder, you need to navigate and locate it